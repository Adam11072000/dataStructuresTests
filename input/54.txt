init 162
mergeSuperPixels 144 51 -6
addImage 222
mergeSuperPixels 222 122 78
getHighestScoredLabel 222 156
addImage 95
deleteImage 95
mergeSuperPixels 222 144 147
deleteImage 222
addImage 250
addImage 250
addImage -2
addImage 1
addImage 4
deleteImage 250
quit