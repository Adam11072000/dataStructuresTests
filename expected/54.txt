init done.
mergeSuperPixels: INVALID_INPUT
addImage: SUCCESS
mergeSuperPixels: SUCCESS
getHighestScoredLabel: FAILURE
addImage: SUCCESS
deleteImage: SUCCESS
mergeSuperPixels: SUCCESS
deleteImage: SUCCESS
addImage: SUCCESS
addImage: FAILURE
addImage: INVALID_INPUT
addImage: SUCCESS
addImage: SUCCESS
deleteImage: SUCCESS
quit done.
